### PR TITLE
[Workers] standard pricing updates

### PR DESCRIPTION
**Pull request by tanushree-sharma**

### CODE REVIEW

File: content/pages/functions/pricing.md [hunk 1]
```diff
@@ -12,18 +12,11 @@ Requests to your Functions are billed as Cloudflare Workers requests. Workers pl
...
-3. In **Overview**, select your Pages project.
-4. Find **Default Usage Model** on the right-side menu and select **Change**.
-
-You can update the Usage Model per project, for both Production and Preview deployments, in the Project Settings page. Like other modifications to your Project Settings, you must redeploy your site in order for this change to take effect. Requests to previous deployments will continue to be billed under the Usage Model the Function was deployed with.
+Pages supports the [Standard Usage Model](workers/platform/pricing/#example-pricing-standard-usage-model).
 
 {{<Aside type="note">}}
 
-For Enterprise customers, please contact your Customer Success Manager for information on pricing.
+Workers Enterprise accounts are billed based on the usage model specified in their contract. To switch to the Standard usage model, reach out to your CSM. Some Workers Enterprise customers maintain the ability to [change usage models](/workers/platform/pricing/#how-to-switch-the-usage-model).
```
Comment (deadlypants1973) [inline content/pages/functions/pricing.md]:
```suggestion
Workers Enterprise accounts are billed based on the usage model specified in their contract. To switch to the Standard usage model, reach out to your Customer Success Manager (CSM). Some Workers Enterprise customers maintain the ability to [change usage models](/workers/platform/pricing/#how-to-switch-the-usage-model).
```

File: content/workers/platform/pricing.md [hunk 2]
```diff
@@ -8,11 +8,13 @@ meta:
 
 # Pricing
 
-{{<Aside type="warning" header="Workers Standard pricing is now available">}}
+{{<Aside type="warning">}}
 
-New, [simplified pricing](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) is available for Workers under a new usage model, called Workers Standard. [Opt-in](https://dash.cloudflare.com/?to=/:account/workers/standard/opt-in) to Workers Standard pricing today.
+All users on the Workers Paid plan have been **automatically migrated** from the Bundled and Unbound usage models to the [Standard usage model](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) on March 1, 2024. 
+
+- To learn what this means for Workers pricing head [here](/workers/platform/pricing/).
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
- To learn what this migration means for Workers pricing, refer to [Pricing](/workers/platform/pricing/). 
```

File: content/workers/platform/pricing.md [hunk 3]
```diff
@@ -8,11 +8,13 @@ meta:
...
 # Pricing
 
-{{<Aside type="warning" header="Workers Standard pricing is now available">}}
+{{<Aside type="warning">}}
 
-New, [simplified pricing](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) is available for Workers under a new usage model, called Workers Standard. [Opt-in](https://dash.cloudflare.com/?to=/:account/workers/standard/opt-in) to Workers Standard pricing today.
+All users on the Workers Paid plan have been **automatically migrated** from the Bundled and Unbound usage models to the [Standard usage model](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) on March 1, 2024. 
+
+- To learn what this means for Workers pricing head [here](/workers/platform/pricing/). 
+- To learn about how this impacts Worker limits head [here](/workers/platform/limits/).
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
- To learn about how this migration impacts Worker limits, refer to [Limits](/workers/platform/limits/). 
```

File: content/workers/platform/pricing.md [hunk 4]
```diff
@@ -30,6 +32,13 @@ All [Pages Functions](/pages/functions/) are billed as Workers. All pricing and
 
 ## Workers
 
+Usage models are settings on your Workers that specify how you are billed for usage, as well as the upper [limits](/workers/platform/limits/#worker-limits) for how many milliseconds of CPU time your Worker can use per invocation.
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
Usage models are settings on your Workers that specify how you are billed for usage, as well as the upper [limits](/workers/platform/limits/#worker-limits) for how many milliseconds of {{<glossary-tooltip term_id="CPU time" link="/workers/glossary/?term=cpu+time">}}CPU time{{</glossary-tooltip>}} your Worker can use per invocation. 
```

File: content/workers/platform/pricing.md [hunk 5]
```diff
@@ -58,15 +67,12 @@ A Worker that serves 100 million requests per month, and uses an average of 7 mi
 
 To prevent accidental runaway bills or denial-of-wallet attacks, configure the maximum amount of CPU time that can be used per invocation by [defining limits in your Worker's `wrangler.toml` file](/workers/wrangler/configuration/#limits), or via the Cloudflare dashboard (Workers & Pages > Select your Worker > Settings > CPU Limits).
 
+If you had a Worker on the Bundled usage model prior to the migration to Standard pricing on March 1, 2024, we've automatically added a 50ms CPU limit on your Worker.
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
If you had a Worker on the Bundled usage model prior to the migration to Standard pricing on March 1, 2024, Cloudflare has automatically added a 50 ms CPU limit on your Worker. 
```

File: content/workers/wrangler/configuration.md [hunk 6]
```diff
@@ -148,6 +140,13 @@ After you have opted into the [Workers Standard](/workers/platform/pricing/#work
 
 {{</definitions>}}
 
+### Usage model
+
+As of March 1, 2024 the [usage model](/workers/platform/pricing/#workers) configured in your Worker's `wrangler.toml` will be ignored. The [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model) usage model applies. 
+
+Some Workers Enterprise customers maintain the ability to change usage models. Your usage model must  be configured through the Cloudflare dashboard by going to **Workers & Pages** > select your Worker > **Settings** > **Usage Model**.
```
Comment (deadlypants1973) [inline content/workers/wrangler/configuration.md]:
```suggestion
Some Workers Enterprise customers maintain the ability to change usage models. Your usage model must be configured through the Cloudflare dashboard by going to **Workers & Pages** > select your Worker > **Settings** > **Usage Model**.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/functions/pricing.md [hunk 7]
```diff
@@ -12,18 +12,11 @@ Requests to your Functions are billed as Cloudflare Workers requests. Workers pl
...
 
-Pages supports both Bundled and Unbound Usage Models. When an account is first upgraded to a Paid Plan, Unbound is the default Usage Model. This default applies only to new projects created after upgrading your plan; existing projects created under the Free Plan will continue to use the Bundled Usage Model. To change the default Usage Model:
-
-1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
-2. In Account Home, select **Workers & Pages**.
-3. In **Overview**, select your Pages project.
-4. Find **Default Usage Model** on the right-side menu and select **Change**.
-
-You can update the Usage Model per project, for both Production and Preview deployments, in the Project Settings page. Like other modifications to your Project Settings, you must redeploy your site in order for this change to take effect. Requests to previous deployments will continue to be billed under the Usage Model the Function was deployed with.
+Pages supports the [Standard Usage Model](workers/platform/pricing/#example-pricing-standard-usage-model).
```
Comment (deadlypants1973) [inline content/pages/functions/pricing.md]:
```suggestion
Pages supports the [Standard usage model](/workers/platform/pricing/#example-pricing-standard-usage-model).
```

**Review by deadlypants1973 — COMMENTED** [review]
broken link

File: content/workers/platform/limits.md [hunk 8]
```diff
@@ -62,7 +62,7 @@ Cloudflare does not enforce response limits, but cache limits for [Cloudflare's
 
 {{<table-wrap>}}
 
-| Feature                     | Free                                       | [Bundled Usage Model](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound Usage Model](/workers/platform/pricing/#example-pricing-unbound-usage-model)|
+| Feature                     | Free                                       | [Bundled Usage Model](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound](/workers/platform/pricing/#example-pricing-unbound-usage-model) and [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model) Usage Model|
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
| Feature                     | Free                                       | [Bundled usage model](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound](/workers/platform/pricing/#example-pricing-unbound-usage-model) and [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model) usage model|
```

**Review by elithrar — COMMENTED** [review]
I know this is existing content, but should we provide 2-3 more examples here?

Specifically, quantify what a "small app", "medium app" and "large app" would look like - so that folks don't assume it's $45/mo out of the box. We've (started to) do that with DO - https://developers.cloudflare.com/durable-objects/platform/pricing/#durable-objects-billing-examples - but could be better: 

image

**Comment by tanushree-sharma** [comment]
@elithrar That's a great idea! Addressing as a follow up PR.

**Pull request merged**